const run = require('./run');

run().catch((_err) => process.exit(1));
